website/docs/cloud-docs/users-teams-organizations/single-sign-on/azure-ad.mdx

```diff
@@ -79,15 +79,22 @@ To edit your Azure SSO configuration settings:
 To configure team management in your Microsoft Azure AD application:
 
 1. Navigate to the single sign-on page.
-1. Edit step 2, "User Attributes & Claims."
-   We recommend naming it "MemberOf", leaving the namespace blank, and potentially sourcing `user.assignedroles` as an easy starting point.
+1. Edit step 2, "User Attributes & Claims."  
+2. Add a new group claim.
+3. In the "Group Claims" side window, select the "Security Groups" radio button.  
+   In the "Source Attribute" field, select either sAMAccountName to pass group names in the SAML or "Group ID" to pass thru the UUIDs for the group. See the related Note for more information on UUIDs.
+   Check the "Customize the name of the group claim" box. Use "MemberOf" in the "Name (required)" field. Leave the namespace field blank.
 
--> **Note:** When Azure AD is configured to use Group Claims, it provides Group UUIDs instead of human readable names in its SAML assertions. We recommend [configuring SSO Team IDs](/cloud-docs/users-teams-organizations/single-sign-on#team-names-and-sso-team-ids) for your Terraform Cloud teams to match these Azure Group UUIDs.
+-> **Note:** If you configure Azure AD to use Group Claims, it provides Group UUIDs instead of human readable names in its SAML assertions. We recommend [configuring SSO Team IDs](/cloud-docs/users-teams-organizations/single-sign-on#team-names-and-sso-team-ids) for your Terraform Cloud teams to match these Azure Group UUIDs.
```

Comment:
```suggestion
-> **Note:** When you configure Azure AD to use Group Claims, it provides Group UUIDs instead of human readable names in its SAML assertions. We recommend [configuring SSO Team IDs](/cloud-docs/users-teams-organizations/single-sign-on#team-names-and-sso-team-ids) for your Terraform Cloud teams to match these Azure Group UUIDs.
```